### ISSUE TITLE
Could not find folder with role 'all'.

### ISSUE THREAD

**Issue by LionelR** [nylas/nylas-mail#3381]
First step with Nylas Mail on Ubuntu 16.04, using a Gmail account the Inbox just shows a "Hooray! you're done" message, I only can browse the Spam folder. Sending a mail seems ok but a message appears :
"Error, Your message successfully sent; however, we had trouble saving your message, "test", to your Sent folder.

Could not find folder with role 'all'.
"

**Comment by aslanides** [comment]
I have this issue as well, running Nylas Mail 2.0.32 on Ubuntu 16.04. Clicking 'Show details' in the error dialog does nothing.

**Comment by soumfilipe** [comment]
Yes, I do have the same problem... Nylas Mail 2.0.32 and Ubuntu 16.04

**Comment by soumfilipe** [comment]
Now I'm using the Windows 10 client and have the same problem. I don't think is a OS problem...
